Softeq.XToolkit.WhiteLabel.Droid/MainApplicationBase.cs

```diff
@@ -76,10 +76,18 @@ protected virtual void StartScopeForIoc()
         {
             var containerBuilder = new ContainerBuilder();
             ConfigureIoc(containerBuilder);
+            RegisterServiceLocator(containerBuilder);
             RegisterInternalServices(containerBuilder);
             Dependencies.IocContainer.StartScope(containerBuilder);
         }
 
+        protected virtual void RegisterServiceLocator(ContainerBuilder builder)
+        {
+            var serviceLocator = new IocContainer();
+            Dependencies.Initialize(serviceLocator);
+            builder.Singleton<IIocContainer>(c => serviceLocator);
```

Comment:
Add builder.Singleton<IIocContainer>(c => serviceLocator).PreserveExistingDefaults(); To avoid breaking changes in existed projects

Reply:
Added + used value from dependencies (in case they were previously initialized)